imsv-docs-astro/src/content/docs/guides/authentication/authentication.mdoc

```diff
@@ -0,0 +1,192 @@
+---
+title: Authentication
+slug: guides/authentication
+description: |
+  How to authenticate with Immersve APIs.
+sidebar:
+  order: 1
+---
+
+Immersve APIs are authenticated with either long-lived API Keys or short-lived
+User Session access tokens. Permissions are controlled by roles, scopes and
+accounts. Access tokens can be renewed with a session refresh token. Read on to
+learn more about the Immersve authentication schemes or try out the {% link
+page="resources/public-sandbox-account" title="sandbox credentials" /%} to
+quickly get started. [Contact us](https://immersve.com/contact/) if you are
+ready to provision credentials for your application.
+
+
+## API Key Authentication
+
+The API Key authentication scheme provides enduring access to Immersve APIs. API
+Key authentication is most suitable when your client integration is running on
+your backend server. This will typically be the case for a {% link
+page="use-cases/custodial-wallets" /%} card issuing solution.
+
+Requests using API Key authentication include the following HTTP headers:
+
+{% table %}
+* Header
+* Description
+---
+* <nobr>x-api-key</nobr>
+* Identifier for the Immersve API key. May be logged.
+---
+* <nobr>x-api-secret</nobr>
+* Secret for the Immersve API key. Should not be logged.
+---
+* <nobr>x-account-id</nobr>
+* Optional target account hint for permission resolution. Defaults to the
+  account that owns the API key.
+{% /table %}
+
+The permissions granted to an API Key are determined by its role. See the Roles
+and Permissions section below for the available roles.
```

Comment:
Could link 'Roles and Permissions' to the page anchor

Reply:
I deliberately left this as non-hyperlinked because internal links are hard to maintain.